### DISCUSSION TITLE
SSL setup

### DISCUSSION THREAD

**Discussion by chiel1112** [oss-apps/split-pro#129] [Q&A] [answered]
So I am quite new to docker and was just curious if there is a guide somewhere to get SSL setup on this container? Thanks!

**Comment by KMKoushik** [comment]
you have to run it behind something like caddy/ngnix. caddy handles ssl for you https://caddyserver.com/docs/

**Reply by alexanderwassbjer** [reply, marked as answer]
I assume any reverse proxy does the job, like traefik or nginx 😀

**Comment by chiel1112** [comment]
Perfect thank you, I got NPM set up and it works great!Met vriendelijke groet,Chiel MeibergOn 27 Oct 2024, at 18:07, Alexander Wassbjer ***@***.***> wrote:﻿
I assume any reverse proxy does the job, like traefik or nginx 😀
